### ISSUE TITLE
Updating Feedback Forms

### ISSUE THREAD

**Issue by ivyquinzel** [coordinape/coordinape#1104]
### **Summary**
We are updating the way we gather feedback to the Typeform platform and moving the location of the help section 

### **Specification**
We are asking for:
- Remove the "Give Feedback" button from the Profile dropdown in the top right corner 
- Remove "Docs" button from the Profile dropdown in the top right corner 
- Add "Give Feedback" CTA to the help button in the bottom right-hand corner 
- Change destination link to [Typeform](https://fe7gssn4rhy.typeform.com/to/nvOUfHKN)
- When a user clicks "Give Feedback," a Typeform survey pops up in a modal and can be dismissed 

[UI Reference](https://www.figma.com/proto/twWLrcUjxP4UD20GOQbufX/Design-Scratchpad?page-id=500%3A22423&node-id=501%3A22591&viewport=485%2C520%2C0.18&scaling=min-zoom&starting-point-node-id=501%3A22591)

### **User Story** 
When I want to report a bug, I want to report that bug easily, so I can get the issue fixed to unblock my work. 

**Acceptance Criteria** 
Scenario: User gets frustrated and wants to give feedback
When: They click on the "?" in the bottom corner 
Then: A modal pop up 
Given: The user clicks "GIVE feedback"
Then: A modal pops up with a survey 
When: The user is finished with the form. They can click out and continue working.
Then: Modal disappears, and the help button collapses

**Comment by ivyquinzel** [comment]
Tagging crab for approval

**Comment by CryptoGraffe** [comment]
@ivyquinzel I'm curious why the "GIVE Feedback" link in the help uses all caps GIVE since it isn't about GIVING it seems kinda confusing to me, it seems like it might be about donating? Also it uses the same icon as send receive GIVE.

**Comment by CryptoGraffe** [comment]
@ivyquinzel is the security concern about the typeform modal? I think it could be ok but it won't work with that standard link. We'll need to use one of their embeds, so I would need access. I have made all the changes requested, except that typeform loads in a new tab. I propose we move forward with this and then do a new ticket for embedding typeform you really think it is necessary.
